Quadriga/src/main/java/edu/asu/spring/quadriga/dto/UploadTransfomationFilesDTO.java

```diff
@@ -0,0 +1,56 @@
+package edu.asu.spring.quadriga.dto;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Column;
+@Entity
+@Table(name = "trns_files")
+public class UploadTransfomationFilesDTO {
```

Comment:
Why is it called "UploadTransformationFilesDTO". It just stores transformation files, right? The name should match what it is, not how it got there.